### PR TITLE
Updated Syntax as per ES6

### LINKED ISSUE #16662: No more vars in Web/API
This is the companion to https://github.com/mdn/content/issues/16614, but for Web/API.

Here's a spreadsheet: https://docs.google.com/spreadsheets/d/1sl6gT8o-7yRPymZprwn6PNBRLQ6KNsKPFILLntsL7gk/edit#gid=0 listing all the pages under Web/API that use `var` .

We should replace usages of `var` with `const` or `let` as appropriate.

If you want to help, please add a comment here *and* request access to the spreadsheet. Once you have access:

- claim some pages by adding your name in the "Fixer" column
- file a PR to fix them. Please add a reference to this issue, but **don't** use "Fixes (this issue)" because that will auto-close this issue when he PR is merged.
- once you have filed a PR, add a link to the PR in the "PR" column
- once the PR is merged, add "Done" in the "Status" column

Note that:

- some of these examples are copied from examples in separate GitHub repositories. In these cases it would be great to update those repos as well, so the examples are the same in both places.
- our policy is to "[use `const` when we can, `let` when we have to](https://github.com/mdn/content/discussions/10389)".
- I'm happy to review other "modern JS" updates, like using `for...of` rather than `for`, and template literals, and arrow functions. But it's not needed at this point.

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
#### Summary

Updated var to const or let as appropriate. 
#### Related issues

Resolving #16662 
#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/htmltextareaelement/index.md [hunk 1]
```diff
@@ -382,7 +382,7 @@ Insert some HTML tags or _smiles_ or any custom text in a textarea.
 
 ```js
 function insertMetachars(sStartTag, sEndTag) {
-  var bDouble = arguments.length > 1, oMsgInput = document.myForm.myTxtArea, nSelStart = oMsgInput.selectionStart, nSelEnd = oMsgInput.selectionEnd, sOldText = oMsgInput.value;
+  let bDouble = arguments.length > 1, oMsgInput = document.myForm.myTxtArea, nSelStart = oMsgInput.selectionStart, nSelEnd = oMsgInput.selectionEnd, sOldText = oMsgInput.value;
```
Comment (PrajwalBorkar) [inline files/en-us/web/api/htmltextareaelement/index.md:385]: This file has some complex code so please review once @wbamberg

Comment (wbamberg) [reply files/en-us/web/api/htmltextareaelement/index.md]: So I hope you don't mind this @PrajwalBorkar , but that example is so hard to understand that I just pushed an update to your PR to refactor it. Now we should be able to review it :).

**Review by wbamberg — APPROVED** [review]
👍 thanks for this PR, @PrajwalBorkar ! Looks good to me.

It looks like you are reusing the branch you used for https://github.com/mdn/content/pull/17107. I think it's better practice to use a different branch for each PR. If that's not your practice, that's fine, but be aware that it might cause you problems.

Also, I'm going to be out of the office for a week from today, so I won't be reviewing PRs. If noone else gets to yours I will have a look when I get back.

**Comment by PrajwalBorkar** [comment]
> 👍 thanks for this PR, @PrajwalBorkar ! Looks good to me.
> 
> It looks like you are reusing the branch you used for #17107. I think it's better practice to use a different branch for each PR. If that's not your practice, that's fine, but be aware that it might cause you problems.
> 
> Also, I'm going to be out of the office for a week from today, so I won't be reviewing PRs. If noone else gets to yours I will have a look when I get back.

>Actually after merging PR I delete the branch there itself. And then create a new branch with the same name as you can see [#17107](https://github.com/mdn/content/pull/17107) here I deleted the branch in last. It's just the name is the same.

>Yeah Okay enjoy the holidays😀

**Pull request merged**